packages/components/src/Link/Link.stories.mdx

```diff
@@ -0,0 +1,68 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Link } from ".";
+
+# Link
+
+<Meta title="Components/Link" component={Link} />
+
+```ts
+import { Link } from "@jobber/components/Link";
+```
+
+<Canvas>
+  <Story
+    name="Link"
+    args={{
+      url: "https://en.wikipedia.org/wiki/Hyperlink",
+      external: true,
+    }}
+  >
+    {args => <Link {...args}>What is a Link anyway?</Link>}
+  </Story>
+</Canvas>
+
+## Props
+
+<ArgsTable of={Link} story="Link" />
+
+## Usage guidelines
+
+The `<Link>` will render an `<a>` tag with the appropriate `href`, `rel` and
+`target` attributes, and included appropriate iconography for external links.
+
+A `<Link>` marked external will always open in a new tab. Set this when linking
+away from Jobber or to other applications or sites across Jobber where this
+behaviour is appropriate (e.g. linking from Jobber Online to the Jobber Help
+Center).
+
+`<Link>` should not be used to submit form data, trigger actions, or alter the
+state of the page. Use a `<Button>` for these purposes.
+
+## Content guidelines
+
+Contents of the link should be textual, and may include some formatting features
+(e.g. italic text for emphasis may be appropriate, changing the inherited
+underline probably isn't). The text can be plain, from a `Typography` component,
+a `format`(`Date|Time|Email`) component or similar. Don't use this component to
+wrap other kinds of UI elements to make them clickable.
+
+## Accessibility
+
+Try to ensure the text of the `<Link>` is descriptive of the destination. Avoid
+using "click here" or "learn more" as the text of the link, and instead use the
+title of the destination page or a short description of the content.
+
+If it's not possible to make the visible text sufficiently descriptive without
+additional context, set the `ariaLabel` prop to a descriptive string for
+screenreaders. Do _not_ set this prop if it will be redundant to the link text.
+
+## Notes
```

Comment:
I think this Notes section could be excluded, as it is more for maintainers than users of the library. Another idea would be to move it to a file in this directory (e.g. `Link.design_notes.txt` or something) to preserve these thoughts for anyone else coming to modify or contribute to the component later. (I will float this idea past the team and see what they think!)